### PR TITLE
Arnab Ghosh hacktober

### PR DESCRIPTION
**Pull request by ghosharnab969**

### CODE REVIEW

File: _hackbars/Arnab_ghosh.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+author: <ARNAB GHOSH>
+---
+
+<!-- Hi folks! this is my first step in open source-->
+
+---
+
+## <!-- Your MESSAGE on climate change -->
+
+![way towards retarding climate change]({{https://pixabay.com/}}/assets/images/<arnab_hackto-ber>.jpg)
```
Comment (tasnimzotder) [inline _hackbars/Arnab_ghosh.md:13]: hey @ghosharnab969, you need to download the image and put on the `assets/images/` directory and link the image here in the `_hackbars/Arnab_ghosh.md` file.

For for more details, you can look at the documentation.

File: _hackbars/Arnab_ghosh.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+layout: hackbar
+title: <First puu request>
+author: <ARNAB GHOSH>
+---
+
+<!-- Hi folks! this is my first step in open source-->
+
+---
+
+## <!-- Your MESSAGE on climate change -->
```
Comment (tasnimzotder) [inline _hackbars/Arnab_ghosh.md:11]: Add __MESSAGE__ on climate change

File: _hackbars/Arnab_ghosh.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
+---
+layout: hackbar
+title: <First puu request>
+author: <ARNAB GHOSH>
+---
+
+<!-- Hi folks! this is my first step in open source-->
```
Comment (tasnimzotder) [inline _hackbars/Arnab_ghosh.md:7]:
```suggestion
---
layout: hackbar
title: First pull request
author: ARNAB GHOSH
---

Hi folks! this is my first step in open source
```

File: _hackbars/Arnab_ghosh.md (hunk 1 above)
Comment (ghosharnab969) [reply _hackbars/Arnab_ghosh.md:13]: ok sure!

**Comment by tasnimzotder** [comment]
We are closing this PR as we did not get any response from you for a while. This PR will be opened if required.

**Pull request closed without merging**